packages/scripts/utils/config.js

```diff
@@ -302,10 +302,71 @@ function getWebpackEntryPoints() {
 	};
 }
 
+/**
+ * Returns the list of paths included in the `render` props by scanning the `block.json` files.
+ *
+ * @return {Array}  The list of all the `render` prop paths included in `block.json` files.
+ */
+function getRenderPropPaths() {
+	// Continue only if the source directory exists.
+	if ( ! hasProjectFile( process.env.WP_SRC_DIRECTORY ) ) {
+		log(
+			chalk.yellow(
+				`Source directory "${ process.env.WP_SRC_DIRECTORY }" was not found. Please confirm there is a "src" directory in the root or the value passed to --webpack-src-dir is correct.`
+			)
+		);
```

Comment:
I think we can skip logging here. It's already handled when detecting entry points where it is essential to process JavaScript files.

Reply:
Thanks! I hadn't realized. I've just deleted it 🙂